Formula/d/dbg-macro.rb

```diff
@@ -0,0 +1,33 @@
+class DbgMacro < Formula
+  desc "Dbg(…) macro for C++"
+  homepage "https://github.com/sharkdp/dbg-macro"
+  url "https://github.com/sharkdp/dbg-macro/archive/refs/tags/v0.5.1.tar.gz"
+  sha256 "fffea75f067c69995853dc790626887788e2c4c9eb0a5a0014a4501d2b6b9909"
+  license "MIT"
+  head "https://github.com/sharkdp/dbg-macro.git", branch: "master"
+
+  depends_on "cmake" => :build
+
+  def install
+    args = std_cmake_args + %w[
+      -DBUILD_TESTING=OFF
```

Comment:
this is part of std_cmake_args

Reply:
std_cmake_args std_cmake_args

Reply:
you dont need this, it default to off in std_cmake_args

Reply:
BUILD_TESTING=OFF removed